drivers/ethernet/eth_nxp_imx_netc_priv.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_ETHERNET_ETH_NXP_IMX_NETC_PRIV_H_
+#define ZEPHYR_DRIVERS_ETHERNET_ETH_NXP_IMX_NETC_PRIV_H_
+
+#include "fsl_netc_endpoint.h"
+#include "fsl_msgintr.h"
+
+/* Buffer and descriptor alignment */
+#define NETC_BD_ALIGN   128
+#define NETC_BUFF_ALIGN 64
```

Comment:
maybe the alignment requirements should be captured in DT

Reply:
This is hardware related. It makes sense to be captured in DT.
Let me try to rework.

Reply:
Double checked RM. This alignment is fixed value required by NETC. 
So, just defining them here as fixed value should be okay.

Reply:
@decsny 
Sorry. I mean the IP block guide. These alignments were for all versions NETC.
Making linux as example, the enetc driver is hard-coding 64 and 128 to use in driver.

THanks.